contracts/ts/__tests__/Maci.test.ts

```diff
@@ -0,0 +1,243 @@
+require('module-alias/register')
+import { genAccounts, genTestAccounts } from '../accounts'
+import { config } from 'maci-config'
+import * as etherlime from 'etherlime-lib'
+
+import { deployAllContracts } from '../deploy'
+
+const accounts = genTestAccounts()
+let deployer
+let maciContract
+describe('MACI', () => {
+    beforeAll(async () => {
+        deployer = new etherlime.JSONRPCPrivateKeyDeployer(
+            accounts[0].privateKey,
+            config.get('chain.url'),
+            {
+                gasLimit: 8800000,
+            },
+        )
+        const contracts = await deployAllContracts(deployer)
+        maciContract = contracts.maciContract
+    })
+
+    it('should be deployed', async () => {
+        expect(maciContract.contractAddress.startsWith('0x')).toBeTruthy()
+    })
+})
+
+//const {
```

Comment:
probably a good idea to either delete the comments or get that code workingl

Reply:
It's a WIP :smile: so we can merge this for it to not be a blocker.

Reply:
:+1: